blocks/extensions/plot.py

```diff
@@ -104,23 +98,25 @@ def do(self, which_callback, *args):
         log = self.main_loop.log
         iteration = log.status['iterations_done']
         i = 0
-        for key, value in log.current_row.items():
```

Comment:
My fault, it's a leftover of a modification that I have then deleted. I will restore it as it was.